### PR TITLE
IDENTITY-2835

### PR DESCRIPTION
**Pull request by DMHP**

### CODE REVIEW

File: components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java [hunk 1]
```diff
@@ -399,10 +399,56 @@ private void handleLogoutRequest(HttpServletRequest request, HttpServletResponse
...
+        for (Enumeration e = request.getHeaderNames(); e.hasMoreElements(); ) {
+            String headerName = e.nextElement().toString();
+            authenticationRequest.addHeader(headerName, request.getHeader(headerName));
+        }
+
+        AuthenticationRequestCacheEntry authRequest = new AuthenticationRequestCacheEntry
+                (authenticationRequest);
+        FrameworkUtils.addAuthenticationRequestToCache(sessionDataKey, authRequest);
+        String queryParams = "?sessionDataKey=" + sessionDataKey
+                + "&" + "type" + "=" + "passivests";
```
Comment (johannnallathamby) [inline components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java]: Use FrameworkConstants.PASSIVE_STS

File: components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java [hunk 2]
```diff
@@ -399,10 +399,56 @@ private void handleLogoutRequest(HttpServletRequest request, HttpServletResponse
...
+        String sessionDataKey = UUIDGenerator.generateUUID();
+        addSessionDataToCache(sessionDataKey, sessionDTO, IdPManagementUtil.getIdleSessionTimeOut
+                (CarbonContext.getThreadLocalCarbonContext().getTenantDomain()));
+        String commonAuthURL = IdentityUtil.getServerURL(FrameworkConstants.COMMONAUTH);
+
+        String selfPath = request.getContextPath();
+        AuthenticationRequest authenticationRequest = new
+                AuthenticationRequest();
+        authenticationRequest.addRequestQueryParam(FrameworkConstants.RequestParams.LOGOUT,
+                new String[]{"true"});
```
Comment (johannnallathamby) [inline components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java]: Can we use Boolean.TRUE here ?

File: components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java (hunk 1 above)
Comment (DMHP) [inline components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java]: Done

File: components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java (hunk 2 above)
Comment (DMHP) [inline components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java]: Yes we can use. Changed the code.

**Pull request merged**